crates/bevy_ecs/src/archetype.rs

```diff
@@ -109,10 +109,12 @@ impl ArchetypeId {
     }
 }
 
+/// Used in AddBundle to track whether components in the bundle are newly
```

Comment:
```suggestion
/// Used in [`AddBundle`] to track whether components in the bundle are newly
```

By adding a doc link here, CI will complain if the referenced struct is renamed, making sure we update our docs.